permissions/pom-structs-parent.yml

```diff
@@ -4,6 +4,7 @@ github: "jenkinsci/structs-plugin"
 paths:
 - "org/jenkins-ci/plugins/structs-parent"
 developers:
+- "@cloudbees-developers"
```

Comment:
Note that this component has been deleted in trunk: https://github.com/jenkinsci/structs-plugin/pull/114

Reply:
Worth removing then I believe, I'll do it.